Sources/Sentry/include/SentryTransaction.h

```diff
@@ -11,8 +11,9 @@ NS_SWIFT_NAME(Transaction)
 SENTRY_NO_INIT
 
 @property (nonatomic, strong) SentryTracer *trace;
+@property (nonatomic, strong) NSArray<id<SentrySpan>> *spans;
 
-- (instancetype)initWithTrace:(SentryTracer *)trace children:(NSArray<id<SentrySpan>> *)children;
+- (instancetype)initWithTrace:(SentryTracer *)trace;
```

Comment:
We were only ever setting this right before capturing the envelope from SentryTracer.finishInternal. Since we also have a reference to that tracer here, we expose the previously private property that held them from the parameter and set it from the same spot that previously init'd the transaction in SentryTracer.finishInternal.